File: TeamF_Api/DAL/Entity/CaffEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TeamF_Api.DAL.Entity
{
    public class CaffEntity
    {
        public int Id { get; set; }
        public virtual User Owner { get; set; }
        public Guid OwnerId { get; set; }
        public string Place { get; set; }
        public virtual ICollection<Img> Images { get; set; }
        public virtual ICollection<Comment> Comments { get; set; }


    }
}

File: compsecfrontend/compSecFrontEnd/src/app/api/models/caff-entity.ts
/* tslint:disable */
/* eslint-disable */
import { Comment } from './comment';
import { Img } from './img';
import { User } from './user';
export interface CaffEntity {
  address?: null | string;
  comments?: null | Array<Comment>;
  creator?: null | string;
  id?: number;
  images?: null | Array<Img>;
  owner?: User;
  ownerId?: string;
}
